desktop-widgets/tab-widgets/maintab.cpp

```diff
@@ -75,6 +75,8 @@ MainTab::MainTab(QWidget *parent) : QTabWidget(parent),
 
 	memset(&displayed_dive, 0, sizeof(displayed_dive));
 
+	modified = false;
```

Comment:
Put this in initializer list, please.

Reply:
updated. Now I am a real C++ programmer ;-)